GitUI/CommandsDialogs/RevisionFileTreeControl.cs

```diff
@@ -404,7 +407,6 @@ private Task ShowGitItemAsync(GitItem gitItem)
                             return ViewGitItemAsync(gitItem);
                         }
 
-                        int? line = FileText.Visible ? FileText.CurrentFileLine : null;
```

Comment:
❓ If the following does not suffice
```
line ??= line = FileText.Visible ? FileText.CurrentFileLine : null;
```
`_requestedLine` must be updated if a line is selected in the FileText.

Reply:
will update this some (mostly to handle repeated RevDiff->FileTree blame), this too
(pushed a tmp/ branch for now)

Reply:
The line is now set directly when there is knowledge to display a file, so it is slightly simpler.